openapi-cli/src/main/java/io/ballerina/openapi/generators/client/FunctionBodyGenerator.java

```diff
@@ -756,7 +757,12 @@ private VariableDeclarationNode getMapForParameters(List<Parameter> parameters,
 
     private boolean checkImportDuplicate(List<ImportDeclarationNode> imports, String module) {
         for (ImportDeclarationNode importModule:imports) {
-            if (importModule.toString().equals("import ballerina/" + module + ";")) {
+            StringBuilder moduleBuilder = new StringBuilder();
+            for (IdentifierToken identifierToken : importModule.moduleName()) {
+                moduleBuilder.append(identifierToken.toString().trim());
+            }
+            if (BALLERINA.equals(importModule.orgName().get().toString().trim().replace("/", "")) &&
```

Comment:
You dont have to do a string replacement to remove the trailing separator token. `io.ballerina.compiler.syntax.tree.ImportOrgNameNode` contains a separate API to get the orgname without the separator.